.github/test_complex.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+DIR=$(dirname $0)
```

Comment:
I think instead of this we should directly invoke `get-input-files.sh`, and same with `test_real`. This avoids code duplication.

Reply:
Got it. I'll change that.